programs/bubblegum/program/src/processor/finalize_tree_with_root.rs

```diff
@@ -169,7 +177,9 @@ pub(crate) fn check_stake<'info>(
         BubblegumError::StakingVoterDiscriminatorMismatch
     );
 
-    let voter: Voter = *bytemuck::from_bytes(&(*voter_bytes.borrow())[8..]);
+    // The Voter has a rather big structure. If we try to deserialize it as other accounts it'll overflow the stack.
```

Comment:
```suggestion
    // The Voter has a rather big structure. If we try to deserialize it on the stack it will be overflowed. Therefore, Box is used to move bytes that have been borrowed to the Heap.
```